src/adapters/bidderFactory.js

```diff
@@ -226,6 +234,19 @@ export function newBidder(spec) {
           onTimelyResponse(spec.code);
           responses.push(resp)
         },
+        /** Extract additional data from a structured response
+         * @param {BidderAuctionResponse} resp
+         */
+        onBidderAuctionResponse: (resp) => {
```

Comment:
I think this should be `onFledgeAuctionConfigs` (and the inputs changed accordingly),  to bring it more in line with the rest of the callbacks here that each take a smaller piece of the data. This suggests that it should process the whole response.

Reply:
We discussed offline, the goal to have a generic chainable callback may have been a little premature.
Added 1297681 to change to `onFledgeAuctionConfigs` callback as suggested.